### PR TITLE
Implement base styles

### PR DESCRIPTION
**Pull request by ronlechiht**

### CODE REVIEW

File: practice-one/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
```
Comment (bannguyen-agilityio) [inline practice-one/.editorconfig]: What does this line mean?

File: practice-one/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Understand HTML/CSS properties and use them correctly
+* Slice from Sketch to HTML/CSS
+* Understand the concepts of Flex
+* Lighthouse performance scoring (greater than 98)
+
+## TECHNICAL
+* HTML5
+* CSS
+
+## TEAM SIZE
```
Comment (bannguyen-agilityio) [inline practice-one/README.md:34]: How to run the app on a local machine?

File: practice-one/src/styles/bases/base.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/* Import font */
```
Comment (bannguyen-agilityio) [inline practice-one/src/styles/bases/base.css]: Do you know `reset css`?

File: practice-one/.editorconfig (hunk 1 above)
Comment (ronlechiht) [reply practice-one/.editorconfig]: i just make a copy on editorconfig.org. Line 11 explain how to apply for multiple files

File: practice-one/README.md (hunk 2 above)
Comment (ronlechiht) [reply practice-one/README.md:34]: Thanks, i will update README.md

File: practice-one/src/styles/bases/base.css (hunk 3 above)
Comment (ronlechiht) [reply practice-one/src/styles/bases/base.css]: yes, i know. but, i think i wrong here. It isn't reset css

**Pull request merged**